### DISCUSSION TITLE
two defects

### DISCUSSION THREAD

**Discussion by yohannes81** [joseph2/vue-slide-unlock#36] [General]
Hi, This beautiful plugin has two defects.

if the cursor is pulled to the left it goes out of the box.

the animation of the button inside a black body causes problems

Is there a way to fix them somehow?

**Comment by j2only** [comment]
Hello, @yohannes81! Thanks for watching and finding the problems.
"_if the cursor is pulled to the left it goes out of the box_" - I fixed this in the latest version 0.5.2
"_the animation of the button inside a black body causes problems_" - I can't understand what we're talking about. Could you demonstrate the problem?

**Reply by yohannes81** [reply]
Hi thanks for replying.
I'll show you two images. on one there is an aesthetic problem.

in the other image I show you an error when I try to install your packages

https://github.com/j2only/vue-slide-unlock/assets/11786960/d7ff0894-3c6e-4699-a564-4f8006d4f1da

![Screenshot 2024-03-05 alle 10 59 14](https://github.com/j2only/vue-slide-unlock/assets/11786960/4c942e99-56b3-426e-a673-d748caa7efaf)

**Reply by j2only** [reply]
I tried to reproduce this problem, but with Chrome 122 the problem is much less obvious. Made some changes to fix the problem. Let me know if the problem remains in version 0.5.3.

https://github.com/j2only/vue-slide-unlock/assets/3985593/838f7771-b70b-4555-8b40-ae46ff9c9ff2

As for installing the package, I apologize, this is my typo in the name of the package. It would be correct - **@j2only/slide-unlock**.

**Reply by yohannes81** [reply]
Thank you
everything seems ok

**Comment by j2only** [comment]
Great! Have a good day.
